src/components/organisms/teacher/OakSearchFilterCheckBox/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./OakSearchFilterCheckBox";
```

Comment:
should we export the props type? unsure if we're doing that elsewhere in the library but feels useful

Reply:
`export * `- Will that not export the prop types ?

Reply:
oh! yeah it will 👀